images/dnsmasq/Dockerfile

```diff
@@ -2,12 +2,11 @@ FROM centos:8
 MAINTAINER CodeReady Container <devtools-cdk@redhat.com>
 
 RUN yum -y install dnsmasq && \
-    yum clean all && \
-    systemctl enable dnsmasq
+    yum clean all
 
 COPY dnsmasq.conf /etc/
 COPY resolv.dnsmasq.conf /etc/
 
 EXPOSE 53/udp
 
-CMD [ "/sbin/init" ]
+ENTRYPOINT ["/usr/sbin/dnsmasq", "-d"]
```

Comment:
From `man dnsmasq`:
```
       -d, --no-daemon
              Debug  mode:  don't fork to the background, don't write a pid file, don't change user id, generate a complete cache dump on receipt on SIGUSR1, log to stderr as well as syslog, don't fork new
              processes to handle TCP queries. Note that this option is for use in debugging only, to stop dnsmasq daemonising in production, use -k.
```
I'd use the long option name fwiw, so `--keep-in-foreground`

Reply:
@cfergeau I thought `-d` make more sense in our case since we are able to collect more logs if it crashed ?

Reply:
But `don't fork new processes to handle TCP queries` might be good point not to use it since there is a lot of request which is going through local dns server, I will update it

Reply:
They say not no use `-d` in production, I'm not going to try to second guess dnsmasq's authors :)